.github/workflows/publish.yml

```diff
@@ -0,0 +1,43 @@
+name: Publish Docker image
+on:
+# Will only run when release is published. 
+  push:
+  release:
+    types: [published]
+  workflow_dispatch:
+
+jobs:
+  publish-images:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2.3.4
+        with:
+          fetch-depth: 0
+
+      - name: Setup Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 14
+
+      - name: create checksum file
+        run: find . -type f -name "*.gradle*" -o -name "gradle-wrapper*" | xargs shasum | sort > checksum.txt && cat checksum.txt
```

Comment:
Would it make sense to publish something of the things like this that we repeat across repos and builds as our own actions?

Reply:
Yup. It makes sense. 

I am thinking of doing something like this: https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/sharing-workflows-with-your-organization
 
This will help us import this standard template across the orgs. 

But even if we want to publish this as separate action, we can do that.